website/blog/2024-10-31-release-1.14.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Podman Desktop 1.14 Release
+description: Podman Desktop 1.14 has been released!
+slug: podman-desktop-release-1.14
+authors: [gastoner]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+<!-- This image link is used for social media previews / thumbnails. Release images are available: https://github.com/containers/podman-desktop-internal/tree/main/release-images -->
+image: /blog/img/podman-desktop-release-1.14/banner.png
+---
+
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.14 Release! 🎉
+
+![podman-desktop-hero-1.14](img/podman-desktop-release-1.14/banner.png)
+
+Podman Desktop 1.14 is now available. [Click here to download it](/downloads)!
+
+This release includes: 
+
+- **Kubernetes improvements with new dashboard**: A new landing screen for Kubernetes has been added with UI changes that will make life with working with Kubernetes clusters easier.
+- **Port forwarding**: This new feature allows users to configure port forwarding in their Kubernetes environment.
+<!--truncate-->
+
+## Release Details
+
+### Kubernetes improvements
+
+
+### Kubernetes improvements with new dashboard
+
+We have updated the Kubernetes dashboard page to give to provide a quick overview of users' Kubernetes clusters, alongside with multiple changes to Kubernetes backend.  
+
+![kubernetes dashboard](img/podman-desktop-release-1.14/kubernetes_dashboard.png)
+
+### Port forwarding
+
+Podman Desktop now supports port forwarding for pods in Kubernetes environments. Port forwarding can be done from pod detail page and then visible in the Port forwarding page. 
+
+![port forwarding](img/podman-desktop-release-1.14/port_forwarding.png)
+
+---
+
+## Community Thank You
```

Comment:
headers are capitalized, but previous headers aren't? small nitpick

Reply:
Do you mean e.g. Kubernetes improvements with a new dashboard => Kubernetes Improvements With A New Dashboard?

Reply:
Yup, I think our release template is a bit messed up too, the headers aren't capitalized.

Reply:
I think IMHO it should just be first letter capitalized, and rest the same, so in this case it would be `Community thank you`

Reply:
created https://github.com/containers/podman-desktop/pull/9703 to fix the template!